src/NzbDrone.Core/Notifications/Webhook/WebhookBase.cs

```diff
@@ -225,5 +229,10 @@ protected WebhookPayload BuildTestPayload()
                 }
             };
         }
+
+        private IEnumerable<string> GetTagLabels(Movie movie)
+        {
+            return movie.Tags?.Select(t => _tagRepository.Get(t)?.Label).Take(5).OrderBy(t => t);
```

Comment:
Should do the Take(5) before the Select so we don't make all the DB calls after the 5

Reply:
Fine by mean, it shouldn't matter here